CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## v1.7.0 (06/15/2020)
+
+**Common**
```

Comment:
Missing is the compaction API exposed in DefaultCache and the compaction thread starting in background once the cache file size exceeded or is equal to the maximum allowed cache size.